weedcoco/tests/importers/mask_data/category_name_chobbitty

```diff
@@ -0,0 +1,3 @@
+FF0000: "weed: broadleaf"
```

Comment:
call this category_name_cobbity.yaml and you can load it with weedcoco.utils.load_json_or_yaml